src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/JDBCEncryptionDecryptionTest.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Microsoft JDBC Driver for SQL Server
+ * 
+ * Copyright(c) Microsoft Corporation All rights reserved.
+ * 
+ * This program is made available under the terms of the MIT License. See the LICENSE file in the project root for more information.
+ */
+package com.microsoft.sqlserver.jdbc.AlwaysEncrypted;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Properties;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+import org.opentest4j.TestAbortedException;
+
+import com.microsoft.sqlserver.jdbc.SQLServerConnection;
+import com.microsoft.sqlserver.jdbc.SQLServerException;
+import com.microsoft.sqlserver.jdbc.SQLServerPreparedStatement;
+import com.microsoft.sqlserver.jdbc.SQLServerStatement;
+import com.microsoft.sqlserver.jdbc.SQLServerStatementColumnEncryptionSetting;
+import com.microsoft.sqlserver.testframework.DBConnection;
+import com.microsoft.sqlserver.testframework.Utils;
+
+
+
+@RunWith(JUnitPlatform.class)
+public class JDBCEncryptionDecryptionTest extends AESetup {
+    private static SQLServerConnection conn = null;
+    private static SQLServerStatement stmt = null;
+    private static SQLServerPreparedStatement pstmt = null;
+    String [] values = {"10"};
+
+    /**
+     * Connect to specified server and close the connection
+     * @throws Exception 
+     * @throws TestAbortedException 
+     */
+    @Test
+    @DisplayName("test connection")
+    public void testNumeric() throws TestAbortedException, Exception {
```

Comment:
I think name of test is `testNumeric` as test creates NumericTable and populate Numeric Values. 

Can you change `displayName` value also?